prow/jobs/jinja/periodics/label_sync.jinja2

```diff
@@ -16,7 +16,7 @@
       - --config=/etc/config/labels.yaml
       - --confirm=true
       - --orgs=aws-controllers-k8s
-      - --token=/etc/github/oauth
+      - --token=/etc/github/token
```

Comment:
Note to self, this flag will soon be deprecated.
@jljaco i see that the label_sync command has new flags (hourly tokens and allowed burst) are these possible to use with this feature? https://github.com/kubernetes/test-infra/blob/master/label_sync/main.go#L152-L153

Reply:
I don't know about hourly tokens limits and allowed burst.  Could be something to investigate, but isn't that outside of the scope of this PR?

This flag is now used to store the GH client secret, app ID, and client ID in a secret here: https://github.com/jljaco/test-infra/blob/a957857002f11ed2cccf66599d0877cf33fe8069/infra/lib/charts/prow-secrets.ts#L62